### PR TITLE
Upgrade to code-gen v1.10.

### PR DESCRIPTION
**Pull request by shrinandj**
However, there were some manual changes required to the fake clientset generated code. The newly generated
code also included a call to object.Watch() but there was no corresponding implementation for it. This code
was manually deleted. It should be safe since this was in the fake clientset code, the build succeeds and
the unit tests complete successfully.

Testing Done:

1. make test passes
2. Built and deployed the sensor-controller. Ran the example sensor. It completed successfully.

Refs #46

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -30,7 +30,7 @@ required = [
 
 [[constraint]]
   name = "k8s.io/code-generator"
-  branch = "release-1.9"
+  branch = "release-1.10"
```
Comment (magaldima) [inline Gopkg.toml]: Do we also need to update the `k8s.io/apimachinery` and `k8s.io/api` dependencies?

Comment (shrinandj) [reply Gopkg.toml]: All of these could've been changed. But eventually they are dependent on github.com/argoproj/argo which is still at `release-1.9`. That cause the code generation to fail if we bump up the version of `k8s.io/apimachinery` and `k8s.io/api`

**Comment by shrinandj** [comment]
The build failed because the docker image used for building argo-events doesn't include `protobufs`. I tried installing protobuf as part of the build workflow itself. However, that failed because installing protobuf requires unzipping a... well zip file. But the docker image doesn't include `unzip` either :( ...

Then, I tried creating an `argo-events` specific build image. But that failed because some of the instructions for installing protobuf on linux are vague.

I am going to try and fix that. Will keep this PR updated about my progress.

**Comment by shrinandj** [comment]
I'll rebase this PR on top of https://github.com/argoproj/argo-events/pull/51 so that it uses the newer image for builds.

**Comment by shrinandj** [comment]
Re-did some basic testing.
- Built the `sensor-controller`
- Ran the sample sensor. It completed successfully.

**Comment by magaldima** [comment]
lgtm!

**Pull request merged**